packages/jellyfish-transaction/src/script/dftx/dftx_loans.ts

```diff
@@ -209,3 +210,29 @@ export class CCreateVault extends ComposableBuffer<CreateVault> {
     ]
   }
 }
+
+/**
+ * DepositToVault DeFi Transaction
+ */
+export interface DepositToVault {
+  vaultId: string // ---------------| 32 bytes, Vault Id
+  from: Script // ------------------| n = VarUInt{1-9 bytes}, + n bytes, Address containing collateral
+  amount: TokenBalanceVarInt // ----| VarUInt{1-9 bytes} for token Id + 8 bytes for amount, Amount of collateral
```

Comment:
maybe `tokenAmount` as a param would make sense instead of`amount`? 

Minor doesn't have to change if disagree